translate/pytest_translate

```diff
@@ -2,63 +2,41 @@
 #
 # Translate a python test to another language.
 #
-import enum
 import sys
-import os
 import argparse
 import ast
-
+import os.path
+import csharpgen
 from asttoirvisitor import AstToIrVisitor
 
-class Language(enum.Enum):
-    PYTHON = 0
-    CSHARP = 1
-
-
-class UnknownFilenameLanguageError(Exception):
-    def __init__(self, filename):
-        self.filename = filename
 
+def main():
+    args = cmd_line_args()
+    extension = os.path.splitext(args.output_file)[1]
+    if extension != '.cs':
+        raise Exception(f"Unsupported file extension '{extension}'")
```

Comment:
Why raise an exception here?  The error isn't a code bug, it's a user error, so they don't need to see a stack trace.
```python
sys.exit("ERROR: unsupported output file extension '{extension}'")
```